Code/Engine/RendererCore/Pipeline/Implementation/Passes/ForwardRenderPass.cpp

```diff
@@ -113,7 +113,7 @@ void ezForwardRenderPass::SetupPermutationVars(const ezRenderViewContext& render
   ezViewRenderMode::GetDebugText(renderViewContext.m_pViewData->m_ViewRenderMode, sDebugText);
   if (!sDebugText.IsEmpty())
   {
-    ezDebugRenderer::Draw2DText(*renderViewContext.m_pViewDebugContext, sDebugText, ezVec2I32(10, 10), ezColor::White);
+    ezDebugRenderer::DrawInfoText(*renderViewContext.m_pViewDebugContext, ezDebugRenderer::ScreenPlacement::TopLeft, "RenderPassDbg", sDebugText);
```

Comment:
I would leave this text always at a fixed place. This text typically tells what a debug render mode does, which would be odd if this would show up somewhere in the middle of the screen.